tools/Makefile

```diff
@@ -77,6 +77,7 @@ endif
 
 openfst-$(OPENFST_VERSION): openfst-$(OPENFST_VERSION).tar.gz
 	tar xozf openfst-$(OPENFST_VERSION).tar.gz
+	[ -f ./extras/openfst-$(OPENFST_VERSION).patch ] && cd openfst-$(OPENFST_VERSION) && patch -p1 < ../extras/openfst-$(OPENFST_VERSION).patch
 
```

Comment:
shouldn't this line begin with a "-" (after the tab), so that the bad return status will be ignored if this version doesn't have the patch file?